products/tpu/api.yaml

```diff
@@ -0,0 +1,155 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Tpu
+display_name: Google Cloud TPU
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://tpu.googleapis.com/v1/
+    default: true
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Node'
+    input: true
+    base_url: projects/{{project}}/locations/{{zone}}/nodes
+    create_url: projects/{{project}}/locations/{{zone}}/nodes?nodeId={{name}}
+    self_link: projects/{{project}}/locations/{{zone}}/nodes/{{name}}
+    description: |
+      A Cloud TPU instance.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/tpu/docs/'
+      api: 'https://cloud.google.com/tpu/docs/reference/rest/'
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_sec: 900
+          update_sec: 900
+          delete_sec: 900
+      result: !ruby/object:Api::Async::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::Async::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::Async::Error
+        path: 'error'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::String # TODO: resourceref?
```

Comment:
Could you elaborate on why it isn't a resourceref?  Probably because the resource in question is under the compute api, right?